### PR TITLE
daitails-popup-window-branche

### PR DESCRIPTION
**Pull request by ClarenceAtim**
I made it such that ,On desktop, when the user clicks on the "see  project" buttons, they will navigate to the the next/previous project details in the popup.

### CODE REVIEW

**Review by shella12 — COMMENTED** [review]
Hi @ClarenceAtim, 

#  Status: Invalidated :red_circle: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

This milestone is a [pair-programming activity](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md#work-and-submission-mode). You have submitted a project, that does not have any contribution from your partner. Please make sure that your partner contributes into your codebase.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: main.js [hunk 1]
```diff
@@ -1,10 +1,297 @@
...
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industrys standard.',
+    tags: ['HTML', 'Bootstrap', 'Ruby'],
+    demoLink: '',
+    codeLink: '',
+  },
+  {
+    main: false,
+    name: 'Data Dashboard Healthcare',
+    image: './images/Last-card.svg',
```
Comment (fobadara) [inline main.js:76]: [REQUIRED] I noticed that when hovering over any of the project cards the card disappears and I see only the `see project` button. This opposes the Figma design template.

image

[REQUIRED] Kindly ensure that there is a detail pop-up when the `see project` button is clicked. Also, ensure that this pop-up has a close button and ensure that it is responsive. These are requirements for this milestone.

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Clean code 
✅ Good job creating the projects section dynamically

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Clean code 
✅ Good job creating the projects section dynamically

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by medaminedev66** [comment]
Hello @ClarenceAtim👋,

_This is an additional review as a result of the second opinion request._

I can see that the pop-up appears as expected after clicking the "See project" button.

Regarding the second requested change, it is not required to implement it at this milestone. Therefore, you can consider it an optional change.

As these were the only reasons that your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approve right away :)

Keep rocking 🚀!

**Comment by medaminedev66** [comment]
Hello @ClarenceAtim👋,

_This is an additional review as a result of the second opinion request._

I can see that the pop-up appears as expected after clicking the "See project" button.

Regarding the second requested change, it is not required to implement it at this milestone. Therefore, you can consider it an optional change.

The only thing that you need to improve is the responsiveness of the pop-design. The pop-up design does not show up correctly on the mobile version. According to project requirements, you should implement both desktop and mobile design as shown in the Figma template.

Your current design:

image


Keep rocking 🚀!

**Review by Shisui6 — APPROVED** [review]
Hi @ClarenceAtim, 

# Approved 🪄
Your project is complete! There is nothing else to say other than... it's time to merge it 💎
Congratulations! 🎉 

## Highlights 🎯
- Popup window implemented ✅


## Optional suggestions 🪶

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**